uniflow/extract/flow/extract_pdf_flow.py

```diff
@@ -27,13 +25,14 @@ def __init__(
             model_config (Dict[str, Any]): Model config.
         """
         super().__init__()
-        self._process_pdf_op = ProcessPDFOp(
-            name="process_pdf_op",
+        self._extract_pdf_op = ExtractPDFOp(
+            name="extract_pdf_op",
             model=PreprocessModel(
-                guided_prompt_template=guided_prompt_template,
+                guided_prompt_template=None,
```

Comment:
You can rebase the latest code. Then, PreprocessModel does not need `guided_prompt_template` argument anymore.